packages/terra-form-select/src/_Frame.jsx

```diff
@@ -228,6 +228,12 @@ class Frame extends React.Component {
    * Handles the blur event.
    */
   handleBlur(event) {
+    // The check for dropdown.contains(activeElement) is necessary to prevent IE11's from closing dropdown on click of scrollbar in certain contexts.
+    if (this.dropdown && (this.dropdown === document.activeElement && this.dropdown.contains(document.activeElement))) {
+      this.focus();
```

Comment:
`this.focus()` throws an error in IE. Focus will need to be called on a ref to the appropriate element. ( `this.select` or `this.input` depending on the variant type )

One concern I have about this approach is the focus callbacks. `onFocus` is a first class prop on the Select. I don't think we'll want the users onFocus callback fired with this approach. 

```<Select onFocus={() => { console.log('focus'); }} />```